### PR TITLE
fix broken link in pg.gemspec

### PR DESCRIPTION
**Pull request by ethanpailes**
just a minor link fix

### CODE REVIEW

File: pg.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
...
-  s.homepage = "https://bitbucket.org/ged/ruby-pg".freeze
+  s.homepage = "https://github.org/ged/ruby-pg".freeze
```
Comment (cbandy) [inline pg.gemspec]: `github.com`

**Comment by ethanpailes** [comment]
Wow, shameful. I'll fix it when I get to a computer.

**Comment by ethanpailes** [comment]
Ok, should be good to go now.

**Comment by larskanis** [comment]
Thank you!

**Pull request merged**